www/Idfa.js

```diff
@@ -2,9 +2,13 @@ var exec = require("cordova/exec");
 var PLUGIN_NAME = "Idfa";
 
 module.exports = {
-    getInfo: function() {
-        return new Promise(function(resolve, reject) {
+    getInfo: function () {
+        return new Promise(function (resolve, reject) {
             exec(resolve, reject, PLUGIN_NAME, "getInfo", []);
+        }).then(function (obj) {
+            obj.trackingTransparencyStatus =
```

Comment:
Let's remove this line. `trackingTransparencyStatus ` doesn't have any sense for Android.

Reply:
done